ksql-engine/src/main/java/io/confluent/ksql/statement/ConfiguredStatement.java

```diff
@@ -30,37 +32,59 @@
   private final PreparedStatement<T> statement;
   private final Map<String, Object> overrides;
   private final KsqlConfig config;
+  private final Checksum checksum;
 
+  public static <S extends Statement> ConfiguredStatement<S> of(
+      final PreparedStatement<S> statement,
+      final Map<String, Object> overrides,
+      final KsqlConfig config,
+      final Checksum checksum
+  ) {
+    return new ConfiguredStatement<>(statement, overrides, config, checksum);
+  }
+
+  @VisibleForTesting
```

Comment:
Since this touches a lot of test files, I will follow-up with a PR that removes this method and explicitly adds `Checksum.none()` to all test locations.